### PR TITLE
Created a new branch 

### PR DESCRIPTION
**Pull request by Otegaa**
Created a new branch for the detail of my work and for contact information to reach me

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <div class="work-tools">
+                    <button>css</button>
+                    <button>html</button>
+                    <button>bootstrap</button>
+                    <button>Ruby</button>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 2]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard
+                  </p>
+                </div>
+                <div class="work-tools work-stack">
+                  <button>html</button>
+                  <button>bootstrap</button>
+                  <button>Ruby</button>
+                </div>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 3]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 4]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 5]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 6]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you use link tags `<ul> <li>` for this. Using correct syntax for your html elements ensures that your code is screen reader friendly.

File: index.html [hunk 7]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="skills">
+          <div class="all-skills">
+            <div class="skills-details">
+              <h1>About Me</h1>
```
Comment (Nyame-Wolf) [inline index.html]: [OPTIONAL] I suggest you change this to any other hx tag `h2 to h6`. your app can only have one H1 element which is the title of your code.

File: index.html [hunk 8]
```diff
@@ -74,6 +74,311 @@ <h2>I'm a software developer</h2>
...
+                >
+              </div>
+            </div>
+            <div class="tech-stack">
+              <div class="stack-heading">
+                <img
+                  src="./assets/icons/Rectangle 55.svg"
+                  alt="language-icon"
+                />
+                <h3>Language</h3>
```
Comment (Nyame-Wolf) [inline index.html:281]: I suggest you change your skills section to a list of list items that is
the language, Framework and skills should also be in `<ul> <li>`

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Otegaa  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same  Figma design ☑️ 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 8 above)
Comment (Otegaa) [reply index.html:281]: @Nyame-Wolf Thank you for your review. I made the changes and now trying to re-submit for review, but cannot see the option for changes requested, just a direct submit. Can I send you the link here and you review it for me? Thank you

Comment (Otegaa) [reply index.html:281]: @Nyame-Wolf Can you kindly review this before the day ends?

Comment (Otegaa) [reply index.html:281]: @Nyame-Wolf https://github.com/Otegaa/setup-and-mobile-first/pull/2

**Pull request merged**